packages/client/components/TeamPrompt/Recurrence/UpdateRecurrenceSettingsModal.tsx

```diff
@@ -128,7 +140,13 @@ export const UpdateRecurrenceSettingsModal = (props: Props) => {
 
     UpdateRecurrenceSettingsMutation(
       atmosphere,
-      {meetingId: meeting.id, recurrenceRule: recurrenceRule?.toString()},
+      {
+        meetingId: meeting.id,
+        recurrenceSettings: {
+          rrule: newRecurrenceSettings.rrule?.toString(),
```

Comment:
-1 we suspect a bug exists somewhere in the client code that allows for a `NaN` interval. let's check for that before sending it to the server so we can provide a friendly message or possibly fix it for them. 
The validation that happens on the GraphQL type isn't handled by the client so if a NaN is triggered the viewer is going to be confused

Reply:
<img width="472" alt="Screenshot 2023-04-04 at 17 58 49" src="https://user-images.githubusercontent.com/1017620/229850694-73df72f7-61c1-470e-b752-53275f2c2c6b.png">

@mattkrick this is how error handling looks like after a 1:1 session with @enriquesanchez

Reply:
awesome! ship it! (pending merge conflicts)